{{cookiecutter.repo_name}}/terraform/modules/aws/main.tf

```diff
@@ -18,6 +18,22 @@ resource "aws_iam_access_key" "key" {
 resource "aws_s3_bucket" "bucket" {
   bucket = "${var.app_name}-${var.environment}"
   acl    = "public-read"
+
+  cors_rule {
```

Comment:
Because this is just the CDN, and these assets will have the same rules that nginx returned, what do you think about just doing * here and letting the nginx response handle it all

Reply:
This is for static assets uploaded into S3, these requests don't hit nginx

Reply:
Oh so media stuff.